models/lease.py

```diff
@@ -8,9 +8,9 @@ class LeaseModel(BaseModel):
     __tablename__ = "lease"
 
     id = db.Column(db.Integer, primary_key=True)
-    name = db.Column(db.String(100))
     propertyID = db.Column(db.Integer, db.ForeignKey('properties.id'), nullable=False)
     tenantID = db.Column(db.Integer, db.ForeignKey('tenants.id'), nullable=False)
     occupants = db.Column(db.Integer)
     dateTimeStart = db.Column(db.DateTime, nullable=False)
     dateTimeEnd = db.Column(db.DateTime, nullable=False)
+    unitNum = db.Column(db.String(10))
```

Comment:
Ooops I thought I made a comment here earlier. Should this be required? `nullable=False`

Reply:
I thought about this when I implemented it. I think making it nullable is fine because not all properties will have units. It could be an ADU type situation.

Reply:
OK sounds good. What is an ADU type situation?

Reply:
nm. I just googled it. Accessory Dwelling Unit.